draft-mouris-cfrg-mastic.md

```diff
@@ -217,10 +219,56 @@ into Mastic in {{vidpf}}. The appendix includes supplementary material:
 {::boilerplate bcp14-tagged}
 
 This document uses the same conventions and definitions of {{Section 2 of
-!VDAF}}. The following functions are as defined therein:
-
-* `gen_rand(len: int) -> bytes`
-* TODO List all functions we use in alphabetical order.
+  !VDAF}}. The following basic functions are as defined therein:
+
+* `bool(val: Any) -> bool` converts a value `val` to a Boolean.
```

Comment:
Do we need to list all these (bool, byte, etc)? My impression was that we wanted just the API functions. @cjpatton WDYT?

Reply:
We use them in this .md. We list basic function like !VDAF does

Reply:
I agree with @jimouris. Yes, we use `bool`'s constructor, but if you understand what `bool` is, then I think think you know what `bool(x)` means since it's a primitive type in python.

Reply:
where do you draw the line on what the reader knows? Why define range() in !VDAF then, and where do you draw the line on what is clear or not?

Reply:
We added that one because someone asked for it.